neo4j_utils/_driver.py

```diff
@@ -830,18 +830,17 @@ def db_status(
 
         name = name or self.current_db
 
-        query = f'SHOW DATABASES WHERE name = "{name}";'
+        query = 'SHOW DATABASES'
```

Comment:
we want to check the status of one specific database, why not to use `WHERE`?

Reply:
In the case that the database we are looking does not exist, this query fails.